scripts/modules/b2b-account/quotes.js

```diff
@@ -21,12 +21,14 @@ define([
     CustomerModels, CartModels, B2BAccountModels, ProductModalViews,
     ProductPicker, ProductModels, WishlistModels, MozuGrid, MozuGridCollection,
     PagingViews, EditableView, QuoteModels) {
-
+        var NAMESTRING = "name cont ";
```

Comment:
Try if we could use 'const' for these variables if we are not going to reassign them anything later in the code

Reply:
const is es6 and is not supported